server/controllers/recipeController.js

```diff
@@ -0,0 +1,34 @@
+const asyncHandler = require("express-async-handler");
+const { getRecipeIngredients } = require("../utils/ingredientUtils");
+const {getAllRecipe, getByName, getByID} = require("../utils/recipeUtils")
+const {Recipe} = require("../models/recipeModel");
+const {Ingredient} = require("../models/ingredientModel");
+
+const viewAllRecipe = asyncHandler( async (req, res) => {
+    const { user } = req.params;
```

Comment:
This will return undefined.
Delete this line and change line 9 `user.id` to `req.user.dataValues.id`

Reply:
Resolved.